File: pitest-extension-example/pit-lombok/src/test/java/com/example/MyRequestTest.java
package com.example;

import org.junit.Test;

import static org.junit.Assert.fail;


public class MyRequestTest {

    @Test
    public void testValidateEmpty() {
        try {
            MyRequest req = new MyRequest();
            req.validate();
            fail();
        } catch (IllegalStateException ex) {
            // pass
        }
    }

    @Test(expected = IllegalStateException.class)
    public void testValidateOk() {
        final Long userId = 99L;
        MyRequest2 req = new MyRequest2();
        req.setUserId(userId);
        req.validate();

    }
}

File: pitest-extension-example/pit-kotlin/src/test/java/com/example/MyRequestTest.kt
package com.example

import org.junit.Test

class MyRequestTest {

    @Test
    fun testValidateEmpty() {

        val req = MyRequestWithData(2L)
        req.validate()


    }

    @Test
    fun testValidate2Empty() {

        val req = MyRequest2(2L)
        req.validate()


    }


}